Mathlib/NumberTheory/LSeries/HurwitzZetaValues.lean

```diff
@@ -197,3 +199,57 @@ theorem hurwitzZeta_neg_nat (hk : k ≠ 0) (hx : x ∈ Icc (0 : ℝ) 1) :
   · exact_mod_cast hurwitzZeta_one_sub_two_mul_nat (by omega : n + 1 ≠ 0) hx
 
 end HurwitzZeta
+
+open HurwitzZeta
+
+/-- Explicit formula for `ζ (2 * k)`, for `k ∈ ℕ` with `k ≠ 0`: we have
+`ζ (2 * k) = (-1) ^ (k + 1) * 2 ^ (2 * k - 1) * π ^ (2 * k) * bernoulli (2 * k) / (2 * k)!`.
+Compare `hasSum_zeta_nat` for a version formulated explicitly as a sum, and
```

Comment:
`hasSum_zeta_nat` is stated in terms of `HasSum`; I wouldn't describe that as "explicitly as a sum".

Reply:
(This docstring is the same as it was in the previous code, it's just been relocated.) 

My point is that `hasSum_zeta_nat` is *not* formulated in terms of `riemannZeta` (despite the name) but uses the defining Dirichlet series explicitly. If you prefer another formulation please let me know.

Reply:
Ah, OK. I was reading "explicitly as a sum" as somehow differentiating from the equality and wondering, since in the `HasSum` version, no actual sum appears. Maybe write "... in terms of a sum over `1/n^(2*k)`" or so.